resources/ark/README.md

```diff
@@ -8,3 +8,61 @@ Ark is a tool to back up and restore Kubernetes resources and persistent volumes
 
 By default, Ark comes with GCP as a backup storage provider and no bucket set. With that configuration, the Ark server deployment scales down to 0 replicas because Ark cannot start without the proper configuration for the backup storage bucket. You can change this by providing proper credentials in the heptio-ark/ark secret and changing the configuration in config/default.
 
+## Usage
+
+> **NOTE:** The steps below assume you have [downloaded](https://github.com/heptio/ark/releases) the ark binary and added it to the environment variable `$PATH` .
+
+> **NOTE:** The steps below assume you have created a lambda function in the production namespace .
+
+
+1. List all namespaces in the kyma cluster.
+
+```
+kubectl --kubeconfig="path_to_kubeconfig" get namespaces
+``` 
+
+2. List all the objects present in the namespace production.
+
+```
+kubectl --kubeconfig="path_to_kubeconfig" get all,serviceinstance,servicebinding,servicebindingusage,function,subscription,api,eventactivation -n production
+```
+
+3. Type `ark` command to become familiar with its usage and flags.
+
+4. Create a backup of the namespace production.
+
+```
+ark --kubeconfig="path_to_kubeconfig" backup create production-backup --include-namespaces production
+```
+
+5. List all the backups
+
+```
+ark --kubeconfig="path_to_kubeconfig" get backups
+```
+
+5. See the description of the created backup `production-backup`.
+
+```
+ark --kubeconfig="path_to_kubeconfig" backup describe production-backup
+```
+
+***ark logs*** can be see it executing the instruction, `kubectl --kubeconfig="path_to_kubeconfig" -n heptio-ark logs deploy/ark` 
```

Comment:
Does this particular line mean that:
- you can see ark logs after executing the command  or
- it is pointing to the `ark-logs` parameter in the command meaning that ark logs are included in it and will be created?

Reply:
It means, 
- you can see ark logs after executing the command

Reply:
I have changed this part, to provide a better explanation